alephclient/tasks/bulkload.py

```diff
@@ -1,14 +1,21 @@
 import logging
+from typing import Optional, Dict
 
+from alephclient.api import AlephAPI
 from alephclient.util import load_config_file
 
 log = logging.getLogger(__name__)
 
 
-def bulk_load(api, mapping_file):
+def bulk_load(api: AlephAPI, mapping_file: str) -> Optional[Dict]:
     data = load_config_file(mapping_file)
+    if not isinstance(data, dict):
+        return
     for foreign_id, config in data.items():
         collection = api.load_collection_by_foreign_id(foreign_id, config)
-        collection_id = collection.get('id')
-        log.info("Bulk mapping collection ID: %s", collection_id)
+        if 'id' not in collection:
+            # NOTE: Raise exception?
+            return None
```

Comment:
I think we should raise an exception there.

Reply:
This is too late. The server should have already thrown an HTTP error code, which I hope has been converted into an exception.

Reply:
Ok, so instead of `collection.get('id')`, we should use `collection['id']`?

Reply:
Can do